src/ixdat/readers/zilien.py

```diff
@@ -162,150 +177,314 @@ def read(self, path_to_file, cls=ECMSMeasurement, name=None, **kwargs):
+            column_headers (list): A list with column names from the current series.
+            data_columns_cut (np.array): A columns cut out from the parsed dataset,
+                that represents the current series.
+
+        Returns:
+            list, DefaultDict[List]: A list of Ixdat series objects and
+            a default dict with standard names for a Mass series.
+        """
+
+        count = self._metadata[f"{series_header}_{series_header}_count"]
+        names_and_units = [
+            self._form_names_and_unit(series_header, column_header)
+            for column_header in column_headers
+        ]
+
+        # Fill Mass aliases
+        aliases = defaultdict(list)
+        for series_name, _, standard_name in names_and_units:
             if standard_name:
                 aliases[standard_name].append(series_name)
 
-            # Form series kwargs and the series
+        # Create Ixdat series
+        column_series = self._create_series_objects(
+            column_headers, names_and_units, data_columns_cut[:count, :]
+        )
+
+        return column_series, aliases
+
+    def _biologic_dataset_part(self, column_headers, data_columns_cut):
+        """Process necessary data for a Biologic dataset part.
+
+        The `experiment_number` and the `technique_number` columns are used only
+        to create an information how to cut the given data part into rows.
+        After that they are not used anymore and no Ixdat series objects
+        are created from them.
+
+        Args:
+            column_headers (list): A list with column names from the current series.
+            data_columns_cut (np.array): A columns cut out from the parsed dataset,
+                that represents the current series.
+
+        Returns:
+            list: A list of Ixdat series objects.
+        """
+
+        count = self._metadata[f"{BIOLOGIC_SERIES_NAME}_{BIOLOGIC_SERIES_NAME}_count"]
+        names_and_units = [
+            self._form_names_and_unit(BIOLOGIC_SERIES_NAME, column_header)
+            for column_header in column_headers
+        ]
+
+        # Split rows according to techniques used according to experiments
+        # Combine the experiment numbers and the technique numbers
+        # in order to create unique identifiers for a successful split
+        exp_nums = data_columns_cut[:count, column_headers.index("experiment_number")]
+        tech_nums = data_columns_cut[:count, column_headers.index("technique_number")]
+        split_vector = exp_nums * 1000 + tech_nums
+        splits = self._get_biologic_splits(split_vector)
+
+        # Create Ixdat series
+        column_series = []
+        for begin, end in splits:
+            column_series += self._create_series_objects(
+                column_headers, names_and_units, data_columns_cut[begin:end, :]
+            )
+
+        return column_series
+
+    def _create_series_objects(self, column_headers, names_and_units, data_rows_cut):
```

Comment:
When I saw this tested in the Zilien reader unit test I realized I do not understand what it is doing. Can you mention in the docstring here that it skips `experiment_number` and `technique_number`, and why?

Reply:
Done. It skips them, because they are not in the MPT files and Ixdat does not create them as well (doesn't have from what). We want to create the same series, not to add extra unnecessary series.